### PR TITLE
Update french translation

### PR DESCRIPTION
**Pull request by Ybbet**
Updated the French translation according to the English file.

### CODE REVIEW

**Review by rejas — CHANGES_REQUESTED** [review]
Could you keep the original order please?

**Comment by Ybbet** [comment]
Yes. On my way. 😉

File: translations/fr.json [hunk 1]
```diff
@@ -29,9 +29,16 @@
 
 	"FEELS": "Ressenti {DEGREE}",
 	"PRECIP_POP": "Probabilité de précipitations",
+	"PRECIP_AMOUNT": "Precipitation amount",
```
Comment (rejas) [inline translations/fr.json:32]: Is that french? ;-) If not, please remove, sicne the english tranlation will be used as a fallback anyway.

**Review by rejas — CHANGES_REQUESTED** [review]

**Comment by Ybbet** [comment]
😅

**Review by rejas — APPROVED** [review]

**Pull request merged**